demo/install.py

```diff
@@ -14,6 +15,12 @@
 MOVIES_SQL_BZ2 = os.path.join(RESOURCES, "movie_collection.sql.bz2")
 
 
+def check_datasets(engine):
+    expect_schemas = [LIBRARY_MANAGEMENT, MOVIE_COLLECTION]
```

Comment:
I assume this will be augmented with new data sets later.

Reply:
Exactly.